src/app/providers.tsx

```diff
@@ -0,0 +1,51 @@
+"use client";
```

Comment:
This file should not be included in this branch as the scope of this PR does not include networking